app/assets/javascripts/custom.js

```diff
@@ -1 +1,7 @@
 document.body.className = 'js-enabled';
+
+$(document).ready(function () {
+    // Use GOV.UK shim-links-with-button-role.js to trigger a link styled to look like a button,
+    // with role="button" when the space key is pressed.
+    GOVUK.shimLinksWithButtonRole.init()
+})
```

Comment:
What this js do?

Is it required if we manually add role to the buttons?

Reply:
It does some JS magic - not sure what exactly - but it is required as well as adding role="button" to any links that look like buttons.   See the link in the Jira ticket - https://github.com/hmrc/accessibility/blob/master/docs/design-and-usability-quick-wins.md#using-links-as-buttons